swap_meet/item.py

```diff
@@ -1,2 +1,45 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__ (self, category="", condition = 0):
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        return f"Hello World!"
+
+    def condition_description(self):
+        if self.condition < 1:
+            return "Poor condition"
+        if self.condition >= 1 and self.condition < 2:
+            return "Worn condition"
+        if self.condition >= 2 and self.condition < 3:
+            return "Good condition"
+        if self.condition >= 3 and self.condition < 4:
+            return "Excellent Condition"
+        if self.condition >=4:
+            return "Brand New Condition"
+        
+# In Wave 2 we will create the `Item` class and the `get_by_category` method.
```

Comment:
Comments are good to leave in committed code if they are adding clarity to some complex logic, explaining usage details, or leaving a note for the next person who might come along and want some context (amongst other reasons). I'd recommend removing them here.